### PR TITLE
Docs: eclipse IDE setup instructions update

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview
- Condense/organize plugin list and setup instructions
- Add notes/screenshots for formatter, import order and cleanup config

## Functional Changes
- documentation only updates

## Manual Testing Performed
- Walked through the documented steps

### Before
screenshot from 2019-01-10 15-49-34

### After
screenshot from 2019-01-10 15-47-50
screenshot from 2019-01-10 15-47-58

### CODE REVIEW

File: docs/dev/setup/ide/eclipse.md [hunk 1]
```diff
@@ -1,32 +1,53 @@
...
-
-  
-Checkstyle:
+# *Eclipse* Setup
+
+
+##  Plugins:
+  - ***Buildship Gradle Integration***
+     1. [Install Plugin](https://marketplace.eclipse.org/content/buildship-gradle-integration)
+  - ***CheckStyle***
```
Comment (ssoloff) [inline docs/dev/setup/ide/eclipse.md]:
```suggestion
  - ***Checkstyle***
```

File: docs/dev/setup/ide/eclipse.md [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
+     1. [Install Plugin](http://eclipse-cs.sourceforge.net)
+     1. Follow: https://checkstyle.org/eclipse-cs/#!/project-setup
+     1. After step 3 "Activate Checkstyle for your project", select the TripleA configuration file instead of "Sun Checks"
+  - ***Lombok*** 
+    1. Download [jar file](https://projectlombok.org/downloads/lombok.jar) and execute it: `java -jar lombok.jar`
+       - [manual install](https://groups.google.com/forum/#!topic/project-lombok/3rVS0eXVl5U)
+         if that does not work
+       - Version to use: 1.16.20
```
Comment (ssoloff) [inline docs/dev/setup/ide/eclipse.md]:
```suggestion
       - Version to use: Same version as Gradle (see `lombok` configuration in the top-level [Gradle buildscript](https://github.com/triplea-game/triplea/blob/master/build.gradle))
```

File: docs/dev/setup/ide/eclipse.md [hunk 3]
```diff
@@ -1,32 +1,53 @@
...
+
+
+## Import Order
+
+
+1. *Project > Java Code Style > Organize Imports*
+1. Click 'Import' and select the import file: [triplea/.eclipse/format/triplea.importorder
+](https://github.com/triplea-game/triplea/blob/master/.eclipse/format/triplea.importorder)
+
+Should look like this when configured: ![screenshot from 2019-01-10 15-10-27
```
Comment (ssoloff) [inline docs/dev/setup/ide/eclipse.md]: Probably want the image on a separate line so it's not pushed to the right by the text.

Comment (DanVanAtta) [reply docs/dev/setup/ide/eclipse.md]: Good suggestion, done: 164ac2864

**Comment by DanVanAtta** [comment]
Thanks for suggestions/fixes @ssoloff , should all be merged or taken care of now.

**Pull request merged**